json-schema/json-schema/src/main/scala/endpoints4s/algebra/JsonSchemas.scala

```diff
@@ -336,7 +336,8 @@ trait JsonSchemas extends TuplesSchemas with PartialInvariantFunctorSyntax {
     *   - Decoders fallback to the `defaultValue` if the field is absent from
     *     the decoded JSON object. They fail if the field is present but has an
     *     invalid value,
-    *   - Encoders always produce the field (even when it has the `defaultValue`),
+    *   - Encoders must emit the field if it is not `defaultValue`, but can
+    *     choose to omit it when it has the `defaultValue`
```

Comment:
We should probably mention the fact that the openapi interpreter is configurable with that regard.

Reply:
The OpenAPI interpreter is not (right now) configurable at all. Should we make it? Only the ujson interpreter is configurable.

Reply:
> The OpenAPI interpreter is not (right now) configurable at all. Should we make it?

No, but we should point users to whatever is configurable.